node/service/src/chain_spec/frequency_rococo.rs

```diff
@@ -28,6 +28,82 @@ pub fn template_session_keys(keys: AuraId) -> frequency_runtime::SessionKeys {
 	frequency_runtime::SessionKeys { aura: keys }
 }
 
+/// Generates the chain spec for a development (no relay)
+pub fn development_config() -> ChainSpec {
+	// Give your base currency a unit name and decimal places
+	let properties =
+		get_properties(FREQUENCY_LOCAL_TOKEN, TOKEN_DECIMALS as u32, Ss58Prefix::get().into());
+
+	ChainSpec::from_genesis(
+		// Name
+		"Frequency Development (No Relay)",
+		// ID
+		"dev",
+		ChainType::Development,
+		move || {
+			testnet_genesis(
+				// initial collators.
+				vec![
+					(
+						get_account_id_from_seed::<sr25519::Public>("Alice"),
+						get_collator_keys_from_seed("Alice"),
+					),
+					(
+						get_account_id_from_seed::<sr25519::Public>("Bob"),
+						get_collator_keys_from_seed("Bob"),
+					),
+				],
+				// Sudo
+				Some(get_account_id_from_seed::<sr25519::Public>("Alice")),
+				// Endowed Accounts
+				vec![
+					get_account_id_from_seed::<sr25519::Public>("Alice"),
+					get_account_id_from_seed::<sr25519::Public>("Bob"),
+					get_account_id_from_seed::<sr25519::Public>("Charlie"),
+					get_account_id_from_seed::<sr25519::Public>("Dave"),
+					get_account_id_from_seed::<sr25519::Public>("Eve"),
+					get_account_id_from_seed::<sr25519::Public>("Ferdie"),
+					get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+					common_runtime::constants::TREASURY_PALLET_ID.into_account_truncating(),
+				],
+				// Council members
+				vec![
+					get_account_id_from_seed::<sr25519::Public>("Alice"),
+					get_account_id_from_seed::<sr25519::Public>("Charlie"),
+					get_account_id_from_seed::<sr25519::Public>("Eve"),
+				],
+				// Technical Committee members
+				vec![
+					get_account_id_from_seed::<sr25519::Public>("Bob"),
+					get_account_id_from_seed::<sr25519::Public>("Dave"),
+				],
+				// ParaId
+				1000.into(),
+			)
+		},
+		// Bootnodes
+		Vec::new(),
+		// Telemetry
+		None,
+		// Protocol ID
+		Some("dev"),
+		// Fork ID
+		None,
+		// Properties
+		Some(properties),
+		// Extensions
+		Extensions {
+			relay_chain: "dev".into(), // You MUST set this to the correct network!
+			para_id: 1000,
+		},
+	)
+}
+
 /// Generates the chain spec for a local testnet
 pub fn local_testnet_config() -> ChainSpec {
```

Comment:
I see `frequency-local` in this. Perhaps that should change to be less confusing.